smithy.yaml

```diff
@@ -1,11 +1,12 @@
 project: dart
 language: dart
 
-# dart 1.19.1
-runner_image: drydock-prod.workiva.org/workiva/smithy-runner-generator:92530
+# dart 1.24.2
+runner_image: drydock-prod.workiva.net/workiva/smithy-runner-generator:203768
 
 script:
-  - pub get
+  - pub get --packages-dir
+  - xvfb-run -s '-screen 0 1024x768x24' pub run dart_dev test --pub-serve --web-compiler=dartdevc -p chrome -p vm
```

Comment:
It's not necessarily required, I added this during testing to point out errors that were specific to the DDC compilation (tests would pass in vm, but not in chrome). Since tests run fast in the VM, I don't think its a big deal to leave it in.